common/utils.py

```diff
@@ -28,6 +28,9 @@
     # like stackdriver logging to happen when running code locally.
     _is_local = False
 
+if os.getenv('FORCE_LOCAL'):
```

Comment:
Sorry could you please add a TODO here to get rid of using http requests for `is_local`?
Thanks!

Reply:
Added and issue: https://github.com/google/fuzzbench/issues/82

Reply:
Added TODO too